<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordAndMoveControls</class>
 <widget class="QWidget" name="RecordAndMoveControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_connect">
     <property name="text">
      <string>Connect to Kuka</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_disconnect">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_recordPosition">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>I want to get robot position!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_position">
     <property name="text">
      <string>Position: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_setAsTarget">
     <property name="text">
      <string>Set as target</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_target">
     <property name="text">
      <string>Target:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_moveToTarget">
     <property name="text">
      <string>Move to target</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
